src/SDKs/Storage/Storage.Tests/Tests/StorageAccountTests.cs

```diff
@@ -1170,12 +1148,27 @@ public void StorageAccountUpdateEncryptionTest()
                 Assert.Equal(true, account.Encryption.Services.File.Enabled);
                 Assert.NotNull(account.Encryption.Services.File.LastEnabledTime);
 
-                // 2. Update storage encryption by enable Blob Encyrption
+                //// 2. Explicitly disable file encryption service.
```

Comment:
We generally discourage commented code. It's best to move this code to a different test and mark it as `Skipped`

Reply:
Good catch.
Fixed.